images/linux/scripts/installers/mongodb.sh

```diff
@@ -6,17 +6,21 @@
 
 # Source the helpers
 source $HELPER_SCRIPTS/os.sh
+source $HELPER_SCRIPTS/install.sh
 
 REPO_URL="https://repo.mongodb.org/apt/ubuntu"
+osLabel=$(getOSVersionLabel)
+toolsetVersion=$(get_toolset_value '.mongodb.default')
+latestVersion=$(curl $REPO_URL/dists/$osLabel/mongodb-org/ | grep $toolsetVersion\.. | awk -F'>|<' '{print $3}' | tail -1)
```

Comment:
maybe slightly change to this to get rid of `\..`?
```suggestion
latestVersion=$(curl $REPO_URL/dists/$osLabel/mongodb-org/ | awk -F'>|<' '{print $3}' | grep "^$toolsetVersion" | tail -1)
```

Reply:
Updated